editor/components/block-settings-menu/block-inspector-button.js

```diff
@@ -46,13 +46,21 @@ export function BlockInspectorButton( {
 export default connect(
 	( state ) => ( {
 		isSidebarOpened: isEditorSidebarOpened( state ),
+		isMobile: ! state.responsive.greaterThan.medium,
```

Comment:
It would be nice to hide this logic behind `isMobile( state )` selector. I think this is a general pattern to always access `state` through selectors. @youknowriad can you confirm?

Reply:
I added an isMobile selector, it simplifies the logic and is more inline with the rest of our code base :)